example/spec/parallel/testAPI/snsRuleSpec.js

```diff
@@ -263,9 +263,10 @@ describe('The SNS-type rule', () => {
     it('adds the new policy and subscription', async () => {
       const { Policy } = await lambda().getPolicy({ FunctionName: consumerName }).promise();
       const { Statement } = JSON.parse(Policy);
-      expect(Statement.length).toEqual(1);
-      expect(Statement[0].Sid).toEqual(`${ruleName}Permission`);
-      expect(await getNumberOfTopicSubscriptions(newTopicArn)).toBe(1);
+      const actualStatement = Statement.find((s) => s.Sid === `${ruleName}Permission`);
+      expect(actualStatement).toBeDefined();
+      expect(actualStatement.Sid).toEqual(`${ruleName}Permission`);
+      expect(await getNumberOfTopicSubscriptions(newTopicArn)).toBeGreaterThan(0);
```

Comment:
I don't have any objections: but are these changes making the test more resilient to any leftover permissions not cleaned up from previous test runs? If so, 💯

Reply:
Correct, this stops the test from expecting exactly 1 policy statement on the lambda and exactly 1 subscription, which both makes it more resilient to failure of previous cleanup, and more parallelization-friendly.